routes/permits/special-uses/non-commercial/index.js

```diff
@@ -19,6 +19,7 @@ var router = express.Router();
 var include = require('include')(__dirname);
 
 var non_commercial = include('controllers/permits/special-uses/non-commercial');
```

Comment:
need to change file name to noncommercial (remove -) for consistency with naming convention